MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -446,6 +454,35 @@ - (nullable NSURL *)buildTokenRequestURLWithAppSecret:(NSString *)appSecret rele
   return components.URL;
 }
 
+- (void)openURLInAuthenticationSessionWith:(NSURL *)url fromClass:(Class)clazz {
+  MSLogDebug([MSDistribute logTag], @"Using SFAuthenticationSession to open URL: %@", url);
+  
+  NSString *callbackUrlScheme = [NSString stringWithFormat:kMSDefaultCustomSchemeFormat, self.appSecret];
+  
+  if (@available(iOS 11.0, *)) {
+    SFAuthenticationSession *session = [[clazz alloc] initWithURL:url callbackURLScheme:callbackUrlScheme completionHandler:^(NSURL *callbackUrl, NSError *error) {
+      
+      self.authenticationSession = nil;
+      MSLogDebug([MSDistribute logTag], @"Called %@ with errror: %@", callbackUrl, error.localizedDescription);
```

Comment:
Can you check and print the log only if `error != nil`? If you want callbackUrl to be in the device log, please add one more log with callback Url.